tools/clang/test/DXC/fd_zs.test

```diff
@@ -1,7 +1,7 @@
 // /Fd plus /Zs
-// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zs /Fd %t.pdb /Fo %t.dxo_fd
+// RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zs /Fd %t.dbg /Fo %t.dxo_fd
 
-// RUN:FileCheck --input-file=%t.pdb %s --check-prefix=PDB && rm %t.pdb
+// RUN:FileCheck --input-file=%t.dbg %s --check-prefix=PDB
```

Comment:
This is to make sure no *.pdb in the directory so the test to test that can work.
If not do this, we have to rm the generated %t.pdb and still hit random fails when tests running at same time.

Reply:
Ah, you're right. `%T` is shared across the directory.

I'm worried about the extent of issues we're having to work around in our tests to avoid problems with the internal build environment. I think we need to figure out a better solution.

Reply:
The name isn't exactly random though is it? Can we duplicate the hash that determines what the name is and check the expected name?

Or perhaps using a custom target like below that uses `%t_pdbdir`? Or did I misunderstand that?

Reply:
Created #5254 to check it by filename.